samples/core/parameterized_tfx_oss/taxi_pipeline_notebook.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Looks good :)</p><p> </p><p>Seems like they can only comment on each block, but not each line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Nice tool. Seems we should do code review from here:</p><p><a href="https://app.reviewnb.com/kubeflow/pipelines/pull/2755/" target="_blank">https://app.reviewnb.com/kubeflow/pipelines/pull/2755/</a></p>